chainer/gradient_check.py

```diff
@@ -370,7 +370,7 @@ class _CheckBackward(object):
 
     def __init__(
             self, func, x_data, y_grad, params, eps, atol, rtol, no_grads,
-            dtype, detect_nondifferentiable):
+            dtype, detect_nondifferentiable, params_to_func):
         if dtype is not None and numpy.dtype(dtype).kind != 'f':
             raise ValueError('`dtype` is allowed only float type')
 
```

Comment:
Let's check the precondition: if `params_to_func` is `True`, `params` must be `ndarray`s.

Reply:
Thanks, fixed.